app/reducers/tabs.js

```diff
@@ -245,10 +245,11 @@ const reopenTab = ( state ) =>
 {
     let { lastTab, lastTabIndex } = getLastClosedTab( state );
 
-    lastTab = { ...lastTab, isClosed: false, closedTime: null };
-    const updatedState = [...state];
+    lastTab = { ...lastTab, isClosed: false, closedTime: null, isActiveTab: true };
```

Comment:
@hunterlester here I think we need to be calling `setActiveTab` with the `lastTabIndex` actually, (which will ensure the previous active tab is unset), and using that as the state.

(For me this isn't working on OSX).

I'd also suggest we add a jest test into the reducer for this case also 👍